Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -261,8 +263,13 @@ def is_valid_type(cls, _type):
             DBotScoreType.CERTIFICATE,
             DBotScoreType.CRYPTOCURRENCY,
             DBotScoreType.EMAIL,
+            DBotScoreType.CUSTOM,
         )
 
+    @classmethod
+    def set_custom_name(cls, name):
+        DBotScoreType.CUSTOM = name
```

Comment:
1. this is not a class method, this is a static method.
2. what if the user wants to use 2 custom indicators?